README.md

```diff
@@ -1 +1,58 @@
-# HELLO-MICROVERSE
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# hello microvese
+
+> hello microverse project.
+
+
+## Built With
+
+- html/css
+
+## Getting Started
+
+**This is a microverse hello project.**
+
+To get a local copy up and running follow these simple example steps.
+
+
+### Setup
+clone this repository or download the Zip folder
+### Install
+git clone <add the link to your repo here>
```

Comment:
- [OPTIONAL] You need your actual url at the place where `<add the link to your repo here>` is written.